onlinejudge/_implementation/utils.py

```diff
@@ -163,7 +163,7 @@ def exec_command(command_str: str, *, stdin: Optional[IO[Any]] = None, input: Op
             answer, _ = proc.communicate(input=input, timeout=timeout)
         except subprocess.TimeoutExpired:
             proc.terminate()
-            answer = None
+            answer = None  # type: ignore
```

Comment:
```suggestion
            answer = None  # type: Optional[bytes]
```

単に ignore するのでなくて、適切な型を指定することで解決したいです。